extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/Scheduled.java

```diff
@@ -131,6 +131,20 @@
      */
     Class<? extends SkipPredicate> skipExecutionIf() default Never.class;
 
+    /**
+     * Defines a period after which the job is considered overdue.
+     * <p>
+     * The value is parsed with {@link Duration#parse(CharSequence)}. However, if an expression starts with a digit, "PT" prefix
+     * is added automatically, so for example, {@code 15m} can be used instead of {@code PT15M} and is parsed as "15 minutes".
+     * Note that the absolute value of the value is always used.
+     * <p>
+     * If the value starts with "&#123;" and ends with "&#125;" the scheduler attempts to find a corresponding config property
```

Comment:
Do we have a test for this?

Reply:
well, it's the same parser as delay & every. 
in OverdueExecutionTest I use the simplified syntax (1s):
https://github.com/quarkusio/quarkus/blob/f95fc935b8b00b1f6e1ce3f33d7711241e42c3b4/extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/OverdueExecutionTest.java#L49-L51

Do you want a more specific test ?

Reply:
I added a test to be sure